### PR TITLE
Property configurator wrapper for Android

### PR DESCRIPTION
**Pull request by annakocheshkova**
Implements `PropertyConfigurator`+`AnalyticsTransmissionTarget.getPropertyConfigurator()` for Android.

### CODE REVIEW

File: Assets/AppCenter/Plugins/AppCenterSDK/Analytics/Android/WrapperPropertyConfiguratorInternal.cs [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+using System.Collections.Generic;
+using UnityEngine;
+using Microsoft.AppCenter.Unity.Analytics.Internal;
+using Microsoft.AppCenter.Unity.Internal.Utility;
+
+namespace Microsoft.AppCenter.Unity.Analytics
+{
+    public class WrapperPropertyConfiguratorInternal
+    {
+        public static void SetAppName(UnityEngine.AndroidJavaObject propertyConfigurator, string appName)
```
Comment (evgeny-pol) [inline Assets/AppCenter/Plugins/AppCenterSDK/Analytics/Android/WrapperPropertyConfiguratorInternal.cs]: Please use just 'AndroidJavaObject' without 'UnityEngine' namespace here and in other similar places

**Review by max-mironov — DISMISSED** [review]

File: Assets/AppCenter/Plugins/AppCenterSDK/Analytics/Blank/WrapperPropertyConfiguratorInternal.cs [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+namespace Microsoft.AppCenter.Unity.Analytics
+{
+#if UNITY_IOS
+    using RawType = System.IntPtr;
+#elif UNITY_ANDROID
+    using RawType = UnityEngine.AndroidJavaObject;
+#else
+    using RawType = System.Object;
+#endif
+    public class WrapperPropertyConfiguratorInternal
```
Comment (evgeny-pol) [inline Assets/AppCenter/Plugins/AppCenterSDK/Analytics/Blank/WrapperPropertyConfiguratorInternal.cs]: Please add newline before this class declaration

File: Assets/AppCenter/Plugins/AppCenterSDK/Analytics/Blank/WrapperPropertyConfiguratorInternal.cs [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+    {
+        public static void SetAppName(RawType propertyConfigurator, string appName)
+        {
+
```
Comment (evgeny-pol) [inline Assets/AppCenter/Plugins/AppCenterSDK/Analytics/Blank/WrapperPropertyConfiguratorInternal.cs]: Please remove unnecessary empty lines

File: Assets/AppCenter/Plugins/AppCenterSDK/Analytics/UWP/WrapperPropertyConfiguratorInternal.cs [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+namespace Microsoft.AppCenter.Unity.Analytics
+{
+#if UNITY_IOS
+    using RawType = System.IntPtr;
+#elif UNITY_ANDROID
+    using RawType = UnityEngine.AndroidJavaObject;
+#else
+    using RawType = System.Object;
+#endif
+    public class WrapperPropertyConfiguratorInternal
```
Comment (evgeny-pol) [inline Assets/AppCenter/Plugins/AppCenterSDK/Analytics/UWP/WrapperPropertyConfiguratorInternal.cs]: Please add newline before class declaration

File: Assets/AppCenter/Plugins/AppCenterSDK/Analytics/UWP/WrapperPropertyConfiguratorInternal.cs [hunk 5]
```diff
@@ -0,0 +1,47 @@
...
+    {
+        public static void SetAppName(IntPtr propertyConfigurator, string appName)
+        {
+
```
Comment (evgeny-pol) [inline Assets/AppCenter/Plugins/AppCenterSDK/Analytics/UWP/WrapperPropertyConfiguratorInternal.cs]: Please remove empty lines

File: Assets/AppCenter/Plugins/AppCenterSDK/Analytics/UWP/WrapperTransmissionTargetInternal.cs [hunk 6]
```diff
@@ -33,6 +33,11 @@ public static Type GetTransmissionTarget(object transmissionTargetParent, string
         {
             return null;
         }
+
+        public static Type GetPropertyConfigurator(IntPtr transmissionTargetParent)
```
Comment (evgeny-pol) [inline Assets/AppCenter/Plugins/AppCenterSDK/Analytics/UWP/WrapperTransmissionTargetInternal.cs]: Shouldn't return value type be WrapperPropertyConfiguratorInternal?

File: Assets/AppCenter/Plugins/AppCenterSDK/Analytics/UWP/WrapperPropertyConfiguratorInternal.cs [hunk 7]
```diff
@@ -0,0 +1,47 @@
...
+#if UNITY_IOS
+    using RawType = System.IntPtr;
+#elif UNITY_ANDROID
+    using RawType = UnityEngine.AndroidJavaObject;
+#else
+    using RawType = System.Object;
+#endif
+    public class WrapperPropertyConfiguratorInternal
+    {
+        public static void SetAppName(IntPtr propertyConfigurator, string appName)
```
Comment (evgeny-pol) [inline Assets/AppCenter/Plugins/AppCenterSDK/Analytics/UWP/WrapperPropertyConfiguratorInternal.cs]: Shouldn't you use 'object' parameter type here and in other methods?

**Review by evgeny-pol — CHANGES_REQUESTED** [review]

**Review by evgeny-pol — APPROVED** [review]

**Pull request merged**